src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -14591,5 +14575,199 @@ static int Test1(out int x)
 
             Assert.Equal("System.Int32", model.GetTypeInfo(yRef).Type.ToTestDisplayString());
         }
+
+        [Fact, WorkItem(13219, "https://github.com/dotnet/roslyn/issues/13219")]
+        public void IndexingDynamic()
```

Comment:
I've marked the place you were missing.

---
In reply to: [75921906](https://github.com/dotnet/roslyn/pull/13274#discussion_r75921906) [](ancestors = 75921906,75763457)